pytorch_blade/src/ltc/include/torch/csrc/lazy/ts_backend/view_ops/as_strided.h

```diff
@@ -0,0 +1,59 @@
+// Copyright 2022 The BladeDISC Authors. All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <torch/csrc/lazy/ts_backend/ts_node.h>
+
+#include <vector>
+
+namespace torch {
+namespace lazy {
+
+class TORCH_API AsStrided : public TsNode {
```

Comment:
newbie question: what's files under `view_ops` directory for? Are they created by us or copied somewhere?

Reply:
Just copy from PyTorch repo, because these header files do not be exposed to PyTorch wheel pacakge. And I think it's not a standard way to collaborate with LTC, because we should maintain these header files for each PyTorch version in the future(>1.12), maybe the implementation just like nvfuse is a better way, we will do more experiments in the future.